src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -303,11 +309,16 @@ export function getInputType(
         return extensibleEnum;
     }
 
-    function getInputTypeForEnum(e: EnumType, addToCollection: boolean = true): InputEnumType {
+    function getInputTypeForEnum(
+        e: EnumType,
+        addToCollection: boolean = true
+    ): InputEnumType {
         let enumType = enums.get(e.name);
         if (!enumType) {
             if (e.members.length == 0) {
-                throw new Error(`Enum type '${e.name}' doesn't define any values.`);
+                throw new Error(
+                    `Enum type '${e.name}' doesn't define any values.`
```

Comment:
Why this formatting changes? Do we have any rules about where to insert new lines?

Reply:
It's prettier rule. There is an constrain of the length of each line.